### PR TITLE
Tim/82 new design

### LINKED ISSUE #82: Card Design Update
What do you guys think of adding labels to the cards with the card type: e.g. Mensa, News, TU Film ...

simulator screen shot 16 04 2017 23 31 59

simulator screen

### PR DESCRIPTION
**Pull request by TG908**
The new Folding Cell Design
Fixes #82

### CODE REVIEW

File: TUM Campus App/CardTableViewCell.swift [hunk 1]
```diff
@@ -7,12 +7,41 @@
...
 
 import UIKit
+import FoldingCell
 import MCSwipeTableViewCell
 
-class CardTableViewCell: MCSwipeTableViewCell {
+class CardTableViewCell: FoldingCell {
+    
     
     func setElement(_ element: DataElement) {
```
Comment (TG908) [inline TUM Campus App/CardTableViewCell.swift]: Can we replace MCSwipeTableViewCell with UITableViewCell?

Comment (mathiasquintero) [inline TUM Campus App/CardTableViewCell.swift]: MCSwipeTableViewCell is there for the swipe gestures. 
Depends on whether we don't want those anymore.

You decide. 'Cause I'm a terrible designer

Comment (TG908) [inline TUM Campus App/CardTableViewCell.swift]: It's also used for table views that don't contain any cards.

Comment (TG908) [inline TUM Campus App/CardTableViewCell.swift]: Also I can no longer use MCSwipeTableViewCell for the cards because they already inherit from FoldingCell

**Comment by TG908** [comment]
Why is the emptyCard not in the cards array? This makes it hard to delete cards and requires extra effort to determine their heights.

**Comment by mathiasquintero** [comment]
There's a difficult Merge Conflict. I can't fix it from here. 
Two files in the xcodeproj have the same ID. 

I suggest:
1. Remove the GradesCardCell file from the project (don't delete it). 
2. Merge Develop into your branch. 
3. Add the GradesCardCell file again

**Comment by TG908** [comment]
@mathiasquintero I still need to wait for the new model. The app will crash if you try to sort the cards because the grades card and books card appear multiple times

**Comment by TG908** [comment]
outdated

**Pull request closed without merging**